app/public/css/styles.css

```diff
@@ -2308,6 +2435,66 @@ button[data-balloon] {
   margin-top: -15px;
 }
 
+/* DELEGATIONS */
+
+.delegations {
+  position: relative;
+  padding-right: 28px;
+}
+
+.delegations::before {
+  position: absolute;
+  content: '';
+  left: 56px;
+  right: 40px;
+  bottom: 40px;
+  height: 40px;
+  background-image: linear-gradient(
+    to top,
+    rgba(255, 255, 255, 1),
+    rgba(255, 255, 255, 0)
+  );
+  z-index: 2;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/22454337/69375834-7038d300-0ca9-11ea-9008-41a09b85a89b.png)
whats this for?

Reply:
Comes from the Transaction History card. Seems to be a design choice to have some whitespace at the bottom of the list.

Reply:
![image](https://user-images.githubusercontent.com/22454337/69418911-d5c5a780-0d1b-11ea-8eb1-deeca22f1e71.png)
you're right. It just seemed weird to me that it covers a little bit of scrollbar which threw me off a little bit.

Reply:
Yes, it seems to be overflowing by a few pixels in all of the lists. Especially visible when you open the page in high resolution.